pkg/util/util.go

```diff
@@ -205,7 +205,7 @@ func TruncateLeft(s string, maxLength int) string {
 	return s
 }
 
-func GetRandomPort() (int , error) {
+func GetRandomPort() (int, error) {
```

Comment:
This has already been fixed on master. I'd suggest you edit the PR description accordingly.